### PR TITLE
Rebased to amazonlinux

### PR DESCRIPTION
**Pull request by mcalr3**
Changed base to Amazon linux, also included symlink for `pip` to `pip3` to ensure compatibility with existing scripts.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 
-COPY ./cdk-packages.txt /cdk/
+COPY ./requirements.txt /cdk/
 COPY ./entrypoint.sh /usr/local/bin/
 
 WORKDIR /cdk
-
-RUN apk -U --no-cache add \
-    bash \
+RUN curl --silent --location https://rpm.nodesource.com/setup_12.x | bash - &&\
```
Comment (StewartW) [inline Dockerfile:9]: Why are we doing this?

File: Dockerfile [hunk 2]
```diff
@@ -1,25 +1,24 @@
-FROM python:3.7.4-alpine3.10
+FROM amazonlinux
 
 RUN mkdir /cdk
 
-COPY ./cdk-packages.txt /cdk/
+COPY ./requirements.txt /cdk/
```
Comment (StewartW) [inline Dockerfile:5]: this doesn't match the name of the file you updated. Ensure they are consistent.

File: Dockerfile (hunk 1 above)
Comment (mcalr3) [reply Dockerfile:9]: this is needed in order to provide a repository for node/npm packages - amazonlinux doesn't have it by default.
The alternative is to use EPEL, but we don't really need to pull down entire EPEL repository just for node.

File: Dockerfile (hunk 2 above)
Comment (mcalr3) [reply Dockerfile:5]: file name updated

**Comment by StewartW** [comment]
This will be exposed under :vnext tag so it can be verified that this works as intended.

**Pull request merged**